assets/js/pages/task_views.js

```diff
@@ -79,6 +79,9 @@
     	$(".view").html(response);
     	window.bindFilter();
       percentileTaskView();
+      if(urlParam('viewer_pane')==1){
```

Comment:
isn't there a way to avoid putting that everywhere ? Like having a global ajax success handler ?